app/views/hyrax/base/_show_actions.html.erb

```diff
@@ -16,20 +16,20 @@
           class: presenter.display_feature_link? ? 'btn btn-default collapse' : 'btn btn-default' %>
     <% end %>
     <% if Hyrax.config.analytics? %>
-      <%= link_to t('.analytics'), presenter.stats_path, id: 'stats', class: 'btn btn-default' %>
+      <%= link_to t('hyrax.base.show_actions.analytics'), presenter.stats_path, id: 'stats', class: 'btn btn-default' %>
```

Comment:
why are these changes necessary?  Doesn't both the before and after resolve to the same key?

Reply:
I agree. But with the implicit scope version (".analytics"), tests fail. Try.